.github/workflows/validate.yml

```diff
@@ -2,33 +2,32 @@ name: validate
 on:
   pull_request:
 
-
 jobs:
   validate:
     runs-on: ubuntu-latest
     steps:
-    - name: checkout
-      uses: actions/checkout@v2
-      with:
-        fetch-depth: 0
-    - name: setup go
-      uses: actions/setup-go@v2
-      with:
-        go-version: 1.16
-    - name: setup python
-      uses: actions/setup-python@v2
-    - name: pre-commit-cache
-      uses: actions/cache@v2
-      env:
-        cache-name: pre-commit-dot-cache
-      with:
-        path: ~/.cache/pre-commit
-        key: ${{ runner.os }}-${{ env.cache-name }}-${{ hashFiles('**/.pre-commit-config.yaml') }}
-    - name: run pre-commit
-      uses: pre-commit/action@v2.0.3
-    - name: run go tests
-      run: make test
-    - name: goreleaser snapshot build
-      uses: goreleaser/goreleaser-action@v2
-      with:
-        args: build --rm-dist --snapshot
+      - name: checkout
```

Comment:
was it intentional to change the indenting here? if there is an agreed-upon standard and you are bringing the file into compliance with that then this is fine, but if it's just your editor then I would revert the whitespace changes